config/locales/en.yml

```diff
@@ -341,6 +341,17 @@ en:
     pagination:
       next: Next
       previous: Prev
+    project_dashboard:
+      activity_title: Latest Activity
+      info_title: Project Information
+      information:
+        created_on: Created on
+        no_description: No description provided for project
+        number_of_automated_workflow_executions: Number of automated workflow executions in project
```

Comment:
**nitpick (blocking):** Can you change this to `Number of automated pipelines configured`

Reply:
Updated in 24d7fa9977a9919ec1c5d1cb43205c9ca9af9db3